internal/server/data/destination.go

```diff
@@ -60,14 +91,24 @@ type GetDestinationOptions struct {
 	ByUniqueID string
 	// ByName instructs GetDestination to return the row matching this name.
 	ByName string
+
+	// FromOrganization is the organization ID of the provider. When set to a
```

Comment:
> FromOrganization is the organization ID of the provider

More of a nit/suggestion: `ByOrganization` for consistency?

Reply:
We have some other variations for these field names (ex: `CreatedBy`, `NotIDs`).

My thinking is that we want something different for `Organization` to make it clear this field works differently from others. `FromOrganization` is an override for the `tx.OrganizationID()`, not simply a new filter being applied to the query.